docs/operators_and_staking/operators.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Operators guide
+sidebar_position: 2
+description: Operators guide
+keywords:
+- Operator
+- Guide
+---
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import styles from '@site/src/pages/index.module.css';
+
+
+:::note
+Currently, the domain chain does not support syncing from other operator nodes; it needs to be deterministically derived from the consensus chain, block by block.
+:::
+
+
+### Build from source
+
+
+The domain operator node is embedded within the `subspace-node` binary, please refer to [Subspace node](https://github.com/subspace/subspace/blob/main/crates/subspace-node/README.md) for how to build from the source.
+
+
+#### Create Operator key:
+An operator needs key pair to participate in Bundle production.
+You can create a key using the following command:
+
+```bash
+target/production/subspace-node key generate --scheme sr25519
+```
+
+
+Back up the key. Take the `Public key (hex)` of the Keypair. The public key is part of the Operator config.
+
+
+#### Insert key to Keystore:
+The key generated above is added to the Keystore so that the Operator node can use it to participate in Bundle production.
+The key is inserted using the following command:
+
+```bash
+target/production/subspace-node key insert \
+--suri "<Secret phrase>" --key-type oper --scheme sr25519 --keystore-path /tmp/keystore
+```
+
+The command above assumes `/tmp/keystore` as the keystore location.
+`suri` is the secret phrase of the Operator key.
+
+
+#### Register Operator:
+The operator needs to register to a domain they want to operate on using `register_operator`. Registration extrinsic requires Operator Config.
```

Comment:
Registration extrinsic requires Operator Config -> The registration extrinsic requires an operator configuration